src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridData</PackageId>
+    <Description></Description>
+    <Version>0.9.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.Management.HybridData</AssemblyName>
+    <PackageTags>Management;Storage;StorSimple;StorSimpleDataManager;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      This is the first public release of the Azure StorSimple Data Manager SDK.
```

Comment:
Is this hybrid or storsimple?

Reply:
HybridDataManager was the old name for the service. The public facing name is StorSimple Data Manager Service

Reply:
Then you should be creating a directory with its appropriate name. What is the RP name now?

Reply:
Hi, The resource provider namespace is "Microsoft.HybridData", but in public documentation, or in releases its mentioned as StorSimple Data Manager. [public doc](https://docs.microsoft.com/en-us/azure/storsimple/storsimple-data-manager-overview). Hence, we use the HybridDataManager. Let us know if this is a problem, in that case, what should be the name.